src/client.ts

```diff
@@ -1,11 +1,26 @@
-import http from 'http';
-
-import { IntegrationProviderAuthenticationError } from '@jupiterone/integration-sdk-core';
-
+import {
+  IntegrationError,
+  IntegrationLogger,
+  IntegrationProviderAPIError,
+  IntegrationProviderAuthenticationError,
+} from '@jupiterone/integration-sdk-core';
+import { createHmac } from 'crypto';
+import * as uuid from 'uuid';
```

Comment:
As a best practice we typically try to avoid `import *`

```suggestion
import { v4 as uuid } from 'uuid';
```